rest/blip_api_crud_test.go

```diff
@@ -1419,6 +1422,9 @@ func TestAccessGrantViaAdminApi(t *testing.T) {
 		connectingPassword: "1234",
 	})
 	defer bt.Close()
+	collection := bt.DatabaseContext().GetSingleDatabaseCollection()
```

Comment:
We want to remove `GetSingleDatabaseCollection` everywhere from the actual `DatabaseContext` code, but we allow it for rest tester. It can't be removed until some more ISGR changes go through...

That said, blip test stores a restTester so if this is needed: `collection := bt.restTester.GetSingleTestDatabaseCollection()`